tools/ci_build/build.py

```diff
@@ -518,6 +518,8 @@ def parse_arguments():
                         help="Disable contrib ops (reduces binary size)")
     parser.add_argument("--disable_ml_ops", action='store_true',
                         help="Disable traditional ML ops (reduces binary size)")
+    parser.add_argument("--disable_sparse_tensors", action='store_true',
+                        help="Disable sparse tensor data types (reduces binary size)")
     parser.add_argument("--disable_rtti", action='store_true', help="Disable RTTI (reduces binary size)")
```

Comment:
Let's keep this to just a cmake variable that has to be specified via `--cmake_extra_args` for now. If more users want to do this we can promote it to being an official arg.